packages/4-xdapp-layer/base/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "private": true,
+  "name": "@wormhole-foundation/sdk-base",
```

Comment:
Is this meant to be the xdapp layer for the `base` chain or the base of the wh sdk?

Reply:
It's the base of the sdk, specifically the top level layer (4-xdapp-layer)

Reply:
We could consider renaming to just `@wormhole-foundation/sdk`